### PR TITLE
IOS-10691 add separators to the sheet informative list and adjusted m…

### PR DESCRIPTION
**Pull request by L-Trujillo26**
> [!WARNING]
> Should a designer need to review/verify this PR?

## 🎟️ **Jira ticket**
https://jira.tid.es/browse/IOS-10691

## 🥅 **What's the goal?**
We need a divider in the informative sheet recycler view to match [design's specification](https://www.figma.com/design/nxToAYk7FGC86TS0mY61TU/%F0%9F%94%B8-Sheet-Specs?node-id=1960-11572&node-type=frame&t=GY2xRf2pOjf1qSDX-0).

## 🚧 **How do we do it?**

1. Introduce a method in ListFragmentView to add rows with optional separators (excluding the last one).
2. Adjust the spacing of InformativeRow according to the specifications.

## 🧪 **How can I verify this?**

- Open mistica's Catalog app.
- Navigate to Sheet, select Children type as Informative and put whatever Icon type you want.

Before:
before

After
after

## 🏑 **AppCenter build**
Version 33.2.1 (1)
https://install.appcenter.ms/orgs/tuenti-organization/apps/mistica-swiftui-ios/distribution_groups/public

### CODE REVIEW

File: Sources/Mistica/Components/Sheet/View/Fragments/List/ListFragmentView.swift [hunk 1]
```diff
@@ -96,9 +96,10 @@ private extension ListFragmentView {
     }
 
     func fillListWithInformativeItems(_ items: [InformativeItem]) {
-        for item in items {
+        for (index, item) in items.enumerated() {
             let rowView = InformativeRow(item: item)
-            stackView.addArrangedSubview(rowView)
+            let isLastItem = index == items.count - 1
+            addRow(rowView, withSeparator: !isLastItem)
```
Comment (DavidMarinCalleja) [inline Sources/Mistica/Components/Sheet/View/Fragments/List/ListFragmentView.swift:102]: One doubt, should the last row have a separator? I mean it is expected:
```
ROW
separator
ROW
separator
ROW
separator
```
or on the contrary it is:
```
ROW
separator
ROW
separator
ROW
```

**Review by DavidMarinCalleja — APPROVED** [review]

**Review by dhidalgofadrique — APPROVED** [review]
goodJobEddieMurphy

**Comment by ieduardogf** [comment]
@yceballost If you are ok with the improvement, we can continue with the PR

**Pull request merged**